src/lib/create.ts

```diff
@@ -12,18 +12,16 @@ export async function create(
   this: Connection,
   query: string
 ): Promise<ContractReceipt> {
-  // TODO: depending on the outcome of some discussions this check to see if address is in
-  //       the passlist can be removed.
-  const authorized = await tablelandCalls.checkAuthorizedList.call(this);
-  if (!authorized) throw new Error("You are not authorized to create a table");
-  // Validation
-
   // This "dryrun" is done to validate that the query statement is considered valid.
   // We check this before minting the token, so the caller won't succeed at minting a token
   // then fail to create the table on the Tableland network
   await tablelandCalls.hash.call(this, query);
 
-  return await registerTable.call(this, query);
+  const txn = await registerTable.call(this, query);
 
   // TODO: we can potentially listen to Execution Tracker here and wait to return
+  //       until we have the receipt, but I'm sure if that makes sense.
```

Comment:
There was a longer internal discussion of waiting to return from this function until the Validator confirms that the transaction was processed.  I left the code to do that commented out so others can potentially investigate